### ISSUE TITLE
i cannot use apple intelligence after i fix the face id

### ISSUE THREAD

**Issue by Kabbutino** [leminlimez/Nugget#115]
hi, when i spoof my iphone to a 16 plus and after i wait apple intelligence finish the download of files, i can't use when i fix the face id, someone can help me plzzz

**Comment by mido22182000** [comment]
مرحبًا، عندما أقوم بتغيير إعدادات الخصوصية في هاتفي iPhone إلى 16 Plus وبعد انتظار أن تنتهي Apple Intelligence من تنزيل الملفات، لا يمكنني استخدامها عندما أقوم بإصلاح Face ID

**Comment by neyarski** [comment]
yo how can i fix the apple intelligence after applying ai apple ?

**Comment by mido22182000** [comment]
Face ID still does not work after installing Apple Intelligence. All attempts have been made, but to no avail. Please solve this problem. Thank you very much.

**Comment by leminlimez** [comment]
try not to open the menu again after enabling

i am looking into this

**Comment by DrJapan** [comment]
Same issue, but iPhone 12 Pro.

**Comment by zp-86** [comment]
Same thing for me (14) but when I set the device to none it leaves it as a 16. if I reset it and then reapply the ai, I have the stuck at downloading problem @leminlimez

**Comment by EthanPany** [comment]
Same here

**Comment by neyarski** [comment]
i found out how it works 
first of all wait until the ai downloaded and works 100% then turn it off from the settings then open nuget app and remove all tweaks then after restarting spoof model device to non and then the ai and the face id will work

**Comment by zp-86** [comment]
Also tried and it still says it is downloading

**Comment by A70411** [comment]
Same here. Set spoof to 'none' then Apple Intelligence not working, and says 'Downloading'. Checked on iPhone Storage and says that Apple Intelligence downloaded 2.56 GB. Unspoofed and spoofed it again, still no Apple Intelligence

**Comment by amaannajam** [comment]
> Same here. Set spoof to 'none' then Apple Intelligence not working, and says 'Downloading'. Checked on iPhone Storage and says that Apple Intelligence downloaded 2.56 GB. Unspoofed and spoofed it again, still no Apple Intelligence

Same problem siri working before but it reverts to old siri but after respoof still saying Apple intelligence downloading it’s already downloaded 2.56gb

**Comment by SerStars** [comment]
> Same thing for me (14) but when I set the device to none it leaves it as a 16. if I reset it and then reapply the ai, I have the stuck at downloading problem @leminlimez

This should be fixed in 4.1

**Comment by neyarski** [comment]
> > Same thing for me (14) but when I set the device to none it leaves it as a 16. if I reset it and then reapply the ai, I have the stuck at downloading problem @leminlimez
> 
> 
> 
> This should be fixed in 4.1

i dont think so cuz its been always the same problem anyway i have the solution and its the 3rd day of using it without any problems if anyone wanna the solution just ask

**Comment by MasterMind-del** [comment]
> hi, when i spoof my iphone to a 16 plus and after i wait apple intelligence finish the download of files, i can't use when i fix the face id, someone can help me plzzz

Follow method #185

**Comment by MasterMind-del** [comment]
> > > Same thing for me (14) but when I set the device to none it leaves it as a 16. if I reset it and then reapply the ai, I have the stuck at downloading problem @leminlimez
> > 
> > 
> > This should be fixed in 4.1
> 
> i dont think so cuz its been always the same problem anyway i have the solution and its the 3rd day of using it without any problems if anyone wanna the solution just ask

Follow method #185

**Comment by MasterMind-del** [comment]
> > > Same thing for me (14) but when I set the device to none it leaves it as a 16. if I reset it and then reapply the ai, I have the stuck at downloading problem @leminlimez
> > 
> > 
> > This should be fixed in 4.1
> 
> i dont think so cuz its been always the same problem anyway i have the solution and its the 3rd day of using it without any problems if anyone wanna the solution just ask

Follow method #185
